crates/sui-replay/src/lib.rs

```diff
@@ -343,12 +367,55 @@ pub async fn execute_replay_command(
             // TODO: clean this up
             Some((0u64, 0u64))
         }
+        ReplayToolCommand::ProfileTransaction {
+            tx_digest,
+            show_effects,
+            diag,
+            executor_version,
+            protocol_version,
+            profile_output,
+        } => {
+            let now = std::time::SystemTime::now()
+                .duration_since(std::time::UNIX_EPOCH)
+                .expect("Error getting system time")
+                .as_nanos();
+            let mut default_name = std::path::PathBuf::from(".");
+            default_name.push(format!("gas_profile_{}_{}.json", tx_digest, now));
```

Comment:
More of a philosophical question: should these profiles be put under a separate (possibly dotted) directory (e.g., a possibly good location may be under `$SUI_HOME`/`~/.sui/`)? I'm just worrying that it may lead to significant file clutter otherwise.

Reply:
Since there is the option to supply the output path, I think its ok. Its been helpful with testing to check cwd!

Reply:
+1 to Laura's take here -- given this is the main thing we're looking for when we run this command, and we can override where it goes with a flag, I think it's fine/ideal to put it in CWD by default (as opposed to the `sui.log` files which I agree are contributing to clutter most of the time).